src/Codeception/Module/WPCLI.php

```diff
@@ -370,7 +370,7 @@ protected function run($userCommand)
             return ['',1];
         }
 
-        if (!empty($this->config['throw']) && $process->getErrorOutput()) {
+        if (!empty($this->config['throw']) && ($process->getExitCode() !== 0)) {
```

Comment:
Btw. it may never go through this IF now since I think:
1) When exit code is not `0` then `WpCliException` will be thrown.
2) It's already handled on line 365 in the catch block.

Reply:
Thanks, good catch.